cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -295,17 +279,17 @@ public void testReconcileCasWhenCustomCertsAreMissingThrows(VertxTestContext con
     }
 
     @Test
-    public void testReconcileCasNoCertsGetGeneratedOutsideRenewalPeriod(VertxTestContext context)
+    public void testReconcileCasNoCertsGetGeneratedOutsideRenewalPeriod(Vertx vertx, VertxTestContext context)
             throws IOException, CertificateException, KeyStoreException, NoSuchAlgorithmException {
-        assertNoCertsGetGeneratedOutsideRenewalPeriod(context, true);
+        assertNoCertsGetGeneratedOutsideRenewalPeriod(vertx, context);
     }
 
-    private void assertNoCertsGetGeneratedOutsideRenewalPeriod(VertxTestContext context, boolean generateCertificateAuthority)
+    private void assertNoCertsGetGeneratedOutsideRenewalPeriod(Vertx vertx, VertxTestContext context)
             throws IOException, CertificateException, KeyStoreException, NoSuchAlgorithmException {
         CertificateAuthority certificateAuthority = new CertificateAuthorityBuilder()
                 .withValidityDays(100)
                 .withRenewalDays(10)
-                .withGenerateCertificateAuthority(generateCertificateAuthority)
+                .withGenerateCertificateAuthority(true)
                 .build();
```

Comment:
@tombentley  raised the question if the default for generateCA being `true` meant we lacked tests where this was `false`. I think that for the test case, we're generating the CA for convenience, the focuus is on the renewal logic. 

There may be a test lacking elsewhere, but I'm not familiar with this space. I hear Tom is a massive fan of certs and cert chains :D

Reply:
@tombentley - true, could I perhaps add another issue to follow up on this? I would have to immerse myself somewhat to write such a test as I'm not yet familiar with this area.

Reply:
@tombentley - look good?